ccvm_simulators/solvers/langevin_solver.py

```diff
@@ -211,6 +249,41 @@ def tune(self, instances, post_processor=None, pump_rate_flag=True, g=0.05):
         #       future consideration
         self.is_tuned = True
 
+    def fpga_energy_max(self, machine_parameters=None):
+        """The wrapper function of calculating the maximum energy consumption of the
+        solver simulating on a fpga machine.
+
+        Args:
+            machine_parameters (dict, optional): Parameters of the. Defaults to None.
```

Comment:
Missing words here, as well. Probably also in the MF solver's version of this function.

Reply:
Thanks for catching them. Done.